[TT-7347] Update Helm Chart docs for tyk-helm-chart release v0.13.2 (#2185)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-oss/ce-helm-chart.md

helm install tyk-ce tyk-helm/tyk-headless -f values.yaml -n tyk
 ```

> Please note that by default, Gateway runs as Deployment and ReplicaCount is 1 because multiple instances of headless gateways won't sync API Definition.

Reviewer: ```
because multiple instances of headless gateways won't sync API Definition.
```
we do not need to specify this part if gw is running as Deployment

Reviewer: on second thought, we may even remove this part completely. this part was added in order to inform users about possible synchronization issues if they are running GW in multiple nodes as DaemonSet which was the default kind for GW. Since we updated the default kind to Deployment, I do not think that we need to document this scenario.

Author: I've left this in so that users will not try to increase the replicaCount, or change it to DaemonSet.

> Please note that by default, Gateway runs as Deployment and ReplicaCount is 1. You should not update this part because multiple instances of headless gateways won't sync API Definition.

Reviewer 2: ```suggestion
Please note that by default, Gateway runs as `Deployment` with `ReplicaCount` is 1. You should not update this part because multiple instances of OSS gateways won't sync the API Definition.
```
